pkg/fixtures/writer_s3.go

```diff
@@ -0,0 +1,108 @@
+package fixtures
+
+import (
+	"context"
+	"github.com/applike/gosoline/pkg/blob"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/aws/aws-sdk-go/aws"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type BlobFixturesSettings struct {
+	BasePath   string
+	ConfigName string
+}
+
+type blobFixtureWriter struct {
+	logger      mon.Logger
+	batchRunner *blob.BatchRunner
+	purger      *blobPurger
+	settings    *BlobFixturesSettings
```

Comment:
actually we only need the BasePath from the settings on the writer, don't we?

Reply:
no we also need the `ConfigName`, passing it to the new store.